antsibull/new_ansible.py

```diff
@@ -84,7 +84,7 @@ def new_ansible_command():
     build_file = BuildFile(build_filename)
     build_file.write(app_ctx.extra['ansible_version'], ansible_base_version, dependencies)
 
-    changelog = ChangelogData.acd(app_ctx.extra['dest_dir'])
+    changelog = ChangelogData.ansible(app_ctx.extra['dest_dir'])
```

Comment:
Interesting that pyre / flake8 weren't reporting this.

Reply:
yeah, IIRC, pyflakes is single file only.  So it would only detect it if ChangelogData was defined in the same file.  pyre only checks functions which have annotations in the function signature (I think annotating the return value is what pyre triggers on).

I like pylint for its thoroughness in regards to checking a whole project.  it's just got so much breadth of tests that it's hard to review its report every time.